docs/developer-docs/latest/setup-deployment-guides/file-structure.md

```diff
@@ -33,6 +33,8 @@ my-project
 
 The default structure of a Strapi project created without the starter CLI looks like the following:
 
+:::: tabs card
+::: tab JavaScript
```

Comment:
Here it makes sense to use VuePress tabs, good job Shaun :-)

But for every other use case, we should use [code groups](https://vuepress.vuejs.org/theme/default-theme-config.html#code-groups-and-code-blocks) to differentiate JS from TS.